services/src/handlers/wfs.rs

```diff
@@ -189,6 +189,19 @@ async fn get_feature<C: Context>(
         .initialize(&execution_context)
         .context(error::Operator)?;
 
+    // handle request and workflow crs matching
+    let spatial_ref = initialized.result_descriptor().spatial_reference();
+    // TODO: use a default spatial reference if it is not set?
+    snafu::ensure!(request.srs_name.is_some(), error::InvalidSpatialReference);
+    // TODO: inject projection Operator?
```

Comment:
```suggestion
    // TODO: inject projection Operator
```
:+1:

Reply:
new task

Reply:
Can you add it as a task somewhere? There are multiple tasks regarding projections (story?). Then, I can close it.

Reply:
https://trello.com/c/ue8BMo64/255-allow-users-to-re-project-data